submissions/lalalilulu/hooli-style_popup/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<header>
+    <nav>
+        <ul class="menu">
+            <li>Mail</li>
```

Comment:
Your popup menu not accessible from keyboard. I can't use your popup using keyboard.